### ISSUE TITLE
DataLossError (see above for traceback): not an sstable (bad magic number)

### ISSUE THREAD

**Issue by DDDoriM1** [tensorflow/tensorflow#23326]
hi, @tensorflow.org.I got a problem in this case. I can run this code in windows, but I can't run this in my Ubuntu 16.04.I got a DataLossError. But the model was found in this case. And this model can run in windows.

I got this Error. So sad.
```
DataLossError (see above for traceback): not an sstable (bad magic number)
	 [[Node: save/RestoreV2 = RestoreV2[dtypes=[DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, ..., DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT], _device="/job:localhost/replica:0/task:0/device:CPU:0"](_arg_save/Const_0_0, save/RestoreV2/tensor_names, save/RestoreV2/shape_and_slices)]]

```
And the code is like this.
```
ckpt_filename = '/home/yangziyuan/SSD-Tensorflow/checkpoints/ssd_300_vgg.ckpt'
isess.run(tf.global_variables_initializer())
saver = tf.train.Saver()
saver.restore(isess.ckpt_filename)
```
But saver.restore() stopped everything.
So anyone has any clue? Thanks for your guys time and patience.

**Comment by harshini-gadige** [comment]
@DDDoriM1  - Hi, please check if [this](https://stackoverflow.com/questions/41048819/how-to-restore-a-model-by-filename-in-tensorflow-r12) helps.

**Comment by DDDoriM1** [comment]
> @DDDoriM1 - Hi, please check if [this](https://stackoverflow.com/questions/41048819/how-to-restore-a-model-by-filename-in-tensorflow-r12) helps.

Hi!!I've checked this link. But it can not solve this problem. I'm using tensorflow 1.11.0. And there is no .meta file in this model. I can run this code by using only .ckpt file in Windows , it can't work in Linux. 

Thank you !!!!!!!!!!

**Comment by JosephPai** [comment]
I got the same problem.
tensorflow-gpu 1.10

**Comment by harshini-gadige** [comment]
@DDDoriM1 - Ok. Can you please fill all the details as per this template so that we can start looking into the issue. We follow a process where we start looking into the bug only if the details are provided as per the template. Thank you!

**Comment by drpngx** [comment]
Are you sure you have the file and it's not truncated? What's the magic that you're getting instead?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 29 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 44 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
We are closing this issue for now due to lack of activity. Please comment if this is still an issue for you. Thanks!

**Comment by YuesongNan** [comment]
I still encounter the same problem. There is no problem for the loading paths. What is the solution?
